cmd/lookoutd/serve.go

```diff
@@ -262,8 +256,13 @@ func (c *ServeCommand) initPoster(conf Config) (lookout.Poster, error) {
 func (c *ServeCommand) initWatcher(conf Config) (lookout.Watcher, error) {
 	switch c.Provider {
 	case github.Provider:
+		urls := make([]string, len(conf.Repositories))
+		for i, repo := range conf.Repositories {
+			urls[i] = repo.URL
+		}
+
 		watcher, err := github.NewWatcher(c.pool, &lookout.WatchOptions{
-			URLs: strings.Split(c.Positional.Repository, ","),
+			URLs: urls,
```

Comment:
Doing a quick read it looks like we don't really use this `WatchOptions.URLs`, only for one log line in `NewWatcher`.
And this `URLs` are taken from the config file, so it will not match the actual URLs we will watch if we use the GitHub App auth, right?

Can we just get rid of it?
The list of repos to watch will be logged as part of the config file for user & token; and with GitHub App auth the logs we already have in `NewClientPoolInstallations`.

Reply:
I removed `WatchOptions` completely